frontend/src/metabase/querying/filters/components/FilterPicker/NumberFilterPicker/NumberFilterPicker.tsx

```diff
@@ -37,26 +37,27 @@ export function NumberFilterPicker({
     valueCount,
     hasMultipleValues,
     isValid,
+    inclusiveOptions,
```

Comment:
Nit: `inclusivityOptions` or even `boundsInclusivity`?
Or even split it up into separate props?